src/bci_build/package/micro/README.md.j2

```diff
@@ -2,12 +2,36 @@
 {% include 'badges.j2' %}
 
 ## Description
-This image is similar to Minimal but without the RPM package manager.
+The `bci-micro` image is even smaller than `bci-minimal` because it also
```

Comment:
Sounds better.

> The `bci-micro` image is smaller than `bci-minimal` because it does not include the RPM package manager.

There's no sentence before describing bci-minimal, it does not feel right to use a comparison tone here.

Reply:
good point, changed.